Jenkinsfile

```diff
@@ -99,22 +99,17 @@ node() {
             }
           }
         }
-      } catch(err) {
-        echo(err)
-        currentBuild.result = "FAILURE"
-
-        if (err.toString().contains('exit code 143')) {
-          currentBuild.result = "ABORTED"
-        }
       } finally {
         stage('Cleanup') {
-          if (params.USE_SAUCELABS) {
-            sh "${env.WORKSPACE}/acceptance-tests/runner/scripts/stop-sc.sh"
-          } else {
-            sh "${env.WORKSPACE}/acceptance-tests/runner/scripts/stop-selenium.sh"
+          catchError(message: 'Suppressing error in Stage: Cleanup') {
```

Comment:
We don't want errors thrown by this stage to mask the errors from the `try` block, so we use `catchError`, which is a step that has similar semantics to `try/catch` but automatically sets the build result and logs the exception if necessary.